net-dns/yaddns/patches/yaddns-1.1.patchset

```diff
@@ -0,0 +1,243 @@
+From ee1290f65761f521bde5fc574a0690e9e7489891 Mon Sep 17 00:00:00 2001
+From: sfanxiang <sfanxiang@gmail.com>
+Date: Mon, 8 Jan 2018 10:10:48 +0000
+Subject: port to Haiku
+
+
+diff --git a/configure.in b/configure.in
+index d8c6668..fd6b42d 100644
+--- a/configure.in
++++ b/configure.in
+@@ -9,6 +9,7 @@ test ".$CFLAGS" = "." && CFLAGS=" "
+ 
+ # programs needed to build/install
+ AC_PROG_CC
++AC_PROG_CXX
+ AC_PROG_INSTALL
+ AC_PROG_RANLIB
+ 
+@@ -66,6 +67,8 @@ if test "x$enable_log_color" = "xyes"; then
+    CFLAGS="$CFLAGS -DENABLE_LOG_COLOR"
+ fi
+ 
++CXXFLAGS=$CFLAGS
++
+ # the generated files
+ AC_CONFIG_FILES([
+ Makefile
+diff --git a/src/Makefile.am b/src/Makefile.am
+index 750ae3a..36e65e9 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -1,3 +1,5 @@
++AM_LDFLAGS = -lbnetapi -lnetwork
```

Comment:
I would prefer using the variable `LIBS`, which allows you to add more link flags without patching. Also, adding `-Wl,--as-needed` would prevents these libraries to be linked when not being used.

```bash
LIBS="-Wl,--as-needed -lbnetapi -lnetwork" runConfigure ./configure
```